modules/bq-log-alerting/README.md

```diff
@@ -0,0 +1,146 @@
+# Log Export: BigQuery Log Alerting
+
+This submodule allows you to configure a BigQuery Log Alerting solution on GCP. It uses the log export created in the root module to create findings in the [Security Command Center](https://cloud.google.com/security-command-center) under a source called "BQ Log Alerts" based on rules defined as views in BigQuery.
+
+## Basic Architecture
+
+### Cloud Logging
+
+The high-level overview of the solution is that [log sinks](https://github.com/terraform-google-modules/terraform-google-log-export) send all Cloud Audit Logs and VPC Flow Logs to [BigQuery](https://github.com/terraform-google-modules/terraform-google-log-export/tree/master/modules/bigquery) located in a centralized logging project.
```

Comment:
nit: since the explanation spans multiple sentences.
```suggestion
The overview of this solution is as follows. [Log sinks](https://github.com/terraform-google-modules/terraform-google-log-export) sends all Cloud Audit Logs and VPC Flow Logs to [BigQuery](https://github.com/terraform-google-modules/terraform-google-log-export/tree/master/modules/bigquery) located in a centralized logging project.
```